docs/examples/sudoers.d/fty_00_base.in

```diff
@@ -52,6 +53,7 @@ User_Alias BIOS_USERS	= %bios-admin,www-data,%bios-infra
 Runas_Alias BIOS_RUNAS	= root
 
 BIOS_USERS ALL=(BIOS_RUNAS) NOPASSWD:BIOS_PROGS
+BIOS_USERS ALL=(BIOS_RUNAS) BIN_TEST
```

Comment:
I think it's better to do ALL=(ALL) BIN_TEST, but your solution should work too.